tools/doc/generate.js

```diff
@@ -21,6 +22,8 @@ args.forEach(function(arg) {
     template = arg.replace(/^--template=/, '');
   } else if (arg.match(/^--node-version=/)) {
     nodeVersion = arg.replace(/^--node-version=/, '');
+  } else if (arg.match(/^\-\-analytics=/)) {
+    analytics = arg.replace(/^\-\-analytics=/, '');
```

Comment:
You're absolutely right, thanks! Replaced with `.startsWith()` as you suggested as well 👍